ui/src/kapacitor/components/TickscriptEditorControls.js

```diff
@@ -19,7 +19,7 @@ const TickscriptEditorControls = ({
     {isNewTickscript ? (
       <TickscriptID onChangeID={onChangeID} id={task.id} />
     ) : (
-      <TickscriptStaticID id={task.name} />
+      <TickscriptStaticID id={task.id} />
```

Comment:
Could you just make sure that downstream, both TickscriptID and TickscriptStaticID components don't use task.id for their HTML id? Otherwise this could introduce some issues.

Reply:
I just checked and neither of the components use an HTML id or key